deku-p/src/core/ledger/ticket_id.mli

```diff
@@ -4,7 +4,7 @@ type ticketer =
 [@@deriving eq, ord, show]
 
 type ticket_id = private Ticket_id of { ticketer : ticketer; data : bytes }
-and t = ticket_id [@@deriving ord, yojson, show]
+and t = ticket_id [@@deriving ord, eq, yojson, show]
```

Comment:
Why do you need this eq?

Reply:
to not do polymorphic eq, but the current vm will be dropped anyways, so i'll remove the eq derivation for now